installation_tips/full_spikeinterface_environment_mac.yml

```diff
@@ -24,15 +24,6 @@ dependencies:
   - compilers
   - pyqt
   - pip:
-#    - PyQt5
     - ephyviewer
-    - neo>=0.10.0
-    - elephant>=0.10.0
-    - probeinterface
-    - MEArec>=1.7.1
-    - spikeinterface[full]
-    - spikeinterface-gui
-    - tridesclous>=1.6.3
-    - phy==2.0b5
-    - herdingspikes
-    - mountainsort4>=1.0.0
```

Comment:
In this yaml env we need to have some sorters and also phy.
The goal for user is to have a full stack working for teaching/training.
So please put back sorters and phy.

Reply:
I'll put these back